redash/tasks/schedule.py

```diff
@@ -21,7 +21,18 @@
 
 logger = logging.getLogger(__name__)
 
-rq_scheduler = Scheduler(
+
+class StatsdRecordingScheduler(Scheduler):
+    """
+    RQ Scheduler Mixin that overrides `enqueue_job` to increment/modify metrics via Statsd
+    """
+
+    def enqueue_job(self, job):
+        super().enqueue_job(job)
+        statsd_client.incr("rq.jobs.created.{}".format(self.queue_name))
```

Comment:
Is this still needed now that we override `enqueue_call`?

Reply:
It was, but it was because I didn't set `queue_class = RedashQueue` and was overriding the wrong fxn in `Queue` (`enqueue_call` instead of `enqueue_job`). Should be fixed now!